src/main/scala/de/unruh/isabelle/pure/ToplevelState.scala

```diff
@@ -34,12 +67,36 @@ object ToplevelState extends MLValueWrapper.Companion[ToplevelState] {
   /** Initializes a new toplevel state based on the theory `theory`. (ML function `Toplevel.theory_toplevel`). */
   def apply(theory: Theory)(implicit isabelle: Isabelle): ToplevelState =
     Ops.theoryToplevel(theory).retrieveNow
+  /** Initializes a new empty toplevel state (based on the 'Pure' theory). */
```

Comment:
Do you know if this is any different from doing `Toplevel(Theory("Pure"))`?

Reply:
The test "theory transition execution" doesn't work with `ToplevelState(Theory("Pure"))`, because it's in Theory mode (as if after "theory .. begin"), not Toplevel. I rephrased the docstrings to suggest one is "in theory" and one just "knows of" theory Pure.

I wrote about the Pure theory in the docstring, because `Toplevel.init_toplevel` calls `Toplevel.init_presentation` which bootstraps Pure, but there's several theory variables inside a ToplevelState and I don't quite understand the differences between them.

Reply:
OK. Still a bit mysterious, but I guess that's the usual Isabelle/ML mystery. :D